installing/validating-an-installation.adoc

```diff
@@ -0,0 +1,62 @@
+[id="validating-an-installation"]
+= Validating an installation
+include::modules/common-attributes.adoc[]
+:context: validating-an-installation
+
+toc::[]
+
+You can check the status of an {product-title} cluster after an installation by following the procedures in this document.
+
+//Reviewing the installation log
+include::modules/reviewing-the-installation-log.adoc[leveloffset=+1]
+
+//Getting cluster version, status, and update details
+include::modules/getting-cluster-version-status-and-update-details.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../support/troubleshooting/troubleshooting-installations.adoc#querying-operator-status-after-installation_troubleshooting-installations[Querying Operator status after installation] for more information about querying Operator status if your installation is still progressing.
+
+* See xref:../support/troubleshooting/troubleshooting-operator-issues.adoc#troubleshooting-operator-issues[Troubleshooting Operator issues] for information about investigating issues with cluster Operators.
```

Comment:
"cluster Operators"? Form the link, it is talking about application operators. And from its naming, this make me feel this item duplicated with the last one, the last link (Querying Operator status after installation) sounds more reasonable for cluster operator trouble shooting.

Reply:
@jianlinliu thank you for your further review.

I have updated this line to state "...investigating issues with cluster Operators". I think it is important to keep this additional reference in place, because it points to a useful assembly on how to address issues with operators. If the reader finds operator problems when validating an installation, it is important that this relevant troubleshooting assembly is found.

I don't think it duplicates the previous additional resource, since that one is a link to a different assembly in the troubleshooting guide, that explicitly talks about querying operator status after installation. My view is that they compliment each other are are both relevant.

"Operator" is always capitalised in our documentation.